client/src/components/Form/Elements/FormBoolean.vue

```diff
@@ -1,24 +1,28 @@
 <template>
-    <b-form-checkbox v-model="currentValue" size="lg" switch />
+    <b-form-checkbox v-model="currentValue" class="no-highlight" switch>
+        {{ label }}
+    </b-form-checkbox>
 </template>
 
 <script>
 export default {
     props: {
         value: {
-            type: Boolean,
             required: true,
         },
     },
     computed: {
         currentValue: {
             get() {
-                return this.value;
+                return ["true", true].includes(this.value);
```

Comment:
In my opinion it's easier to comprehend than array comparison. But it's ofc not critical  
```suggestion
                return JSON.parse(this.value);
```

Reply:
There are both reasonable but I prefer checking the array since it is explicit. I think we should avoid parsing the JSON unless necessary.